bitnami/moodle/templates/_helpers.tpl

```diff
@@ -142,3 +142,14 @@ Return the MariaDB Secret Name
     {{- printf "%s-%s" (include "common.names.fullname" .) "externaldb" -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return the Moodle Secret Name
+*/}}
+{{- define "moodle.secretName" -}}
```

Comment:
This helper already exists, please take a look to the line 70: https://github.com/bitnami/charts/blob/ee19ef71e5f0522013e98c3bae7ed47ce0362920/bitnami/moodle/templates/_helpers.tpl#L67-L72

Reply:
Une right , did not see , will remove my patch on helper.txt

Reply:
i remove the old function and use the standard'one